### PR TITLE
Update README to highlight deployment issue with CocoaPods

### PR DESCRIPTION
**Pull request by SagarSDagdu**
### Changes
This PR updates the README file to update instructions regarding CocoaPods installation

### Pull Request Checklist

- [x] CONTRIBUTING: You have read https://github.com/groue/GRDB.swift/blob/master/CONTRIBUTING.md
- [x] BRANCH: This pull request is submitted against the `development` branch.
- [x] DOCUMENTATION: Inline documentation has been updated.
- [x] DOCUMENTATION: README.md or another dedicated guide has been updated.
- [x] TESTS: Changes are tested.
- [x] TESTS: The `make smokeTest` terminal command runs without failure.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -348,17 +359,24 @@ See [Custom SQLite builds](Documentation/CustomSQLiteBuilds.md) for the installa
...
+# Can't use semantic versioning due to https://github.com/CocoaPods/CocoaPods/issues/11839
+# Replace the tag with the tag that you want to use.
+pod 'GRDB.swift', git: 'https://github.com/groue/GRDB.swift.git', tag: 'v6.29.0' 
+```
+Once CocoaPods resolves this issue and allows deployment of new GRDB versions, users will be able to return to the standard installation method described at the beginning of this section.
 
-> **Note**: Linux is not currently supported.
->
-> **Warning**: Due to an Xcode bug, you will get "No such module 'CSQLite'" errors when you want to embed the GRDB package in other targets than the main application (watch extensions, for example). UI and Unit testing targets are OK, though. See [#642](https://github.com/groue/GRDB.swift/issues/642#issuecomment-575994093) for more information.
+GRDB can be installed as a framework, or a static library.
```
Comment (groue) [inline README.md:379]: I'd move this line back to the top, below the `pod 'GRDB.swift'` line.

File: README.md [hunk 2]
```diff
@@ -348,17 +359,24 @@ See [Custom SQLite builds](Documentation/CustomSQLiteBuilds.md) for the installa
 pod 'GRDB.swift'
 ```
 
-GRDB can be installed as a framework, or a static library.
-
-## Swift Package Manager
+### ⚠️ Important Note for CocoaPods installation
```
Comment (groue) [inline README.md:362]: Good, but it's a little too big.

```suggestion
**Important note for CocoaPods installation**
```

File: README.md [hunk 3]
```diff
@@ -348,17 +359,24 @@ See [Custom SQLite builds](Documentation/CustomSQLiteBuilds.md) for the installa
...
+```ruby
+# Can't use semantic versioning due to https://github.com/CocoaPods/CocoaPods/issues/11839
+pod 'GRDB.swift', git: 'https://github.com/groue/GRDB.swift.git', branch: 'GRDB6'
+```
+- Depend on a specific version explicitly (Replace the tag with the version you want to use):
+```ruby
+# Can't use semantic versioning due to https://github.com/CocoaPods/CocoaPods/issues/11839
+# Replace the tag with the tag that you want to use.
+pod 'GRDB.swift', git: 'https://github.com/groue/GRDB.swift.git', tag: 'v6.29.0' 
+```
```
Comment (groue) [inline README.md:376]: We miss indentation on code blocks. They do not look attached to their bullet points.

File: README.md [hunk 4]
```diff
@@ -348,17 +359,24 @@ See [Custom SQLite builds](Documentation/CustomSQLiteBuilds.md) for the installa
...
+Once CocoaPods resolves this issue and allows deployment of new GRDB versions, users will be able to return to the standard installation method described at the beginning of this section.
```
Comment (groue) [inline README.md:377]: Please remove this sentence, which has not its place in public documentation. I would not be happy to read it if I were a CocoaPods maintainer.

File: README.md [hunk 5]
```diff
@@ -348,17 +359,24 @@ See [Custom SQLite builds](Documentation/CustomSQLiteBuilds.md) for the installa
 pod 'GRDB.swift'
 ```
 
-GRDB can be installed as a framework, or a static library.
-
-## Swift Package Manager
+### ⚠️ Important Note for CocoaPods installation
 
-The [Swift Package Manager](https://swift.org/package-manager/) automates the distribution of Swift code. To use GRDB with SPM, add a dependency to `https://github.com/groue/GRDB.swift.git`
+Due to an [issue](https://github.com/CocoaPods/CocoaPods/issues/11839) in CocoaPods, it is currently not possible to deploy new versions of GRDB to CocoaPods. The last version available on CocoaPods is 6.24.1. If you want to install later versions of GRDB using CocoaPods, please use one of the following workarounds:
```
Comment (groue) [inline README.md:364]:
```suggestion
Due to an [issue](https://github.com/CocoaPods/CocoaPods/issues/11839) in CocoaPods, it is currently not possible to deploy new versions of GRDB to CocoaPods. The last version available on CocoaPods is 6.24.1. To install later versions of GRDB using CocoaPods, use one of the following workarounds:
```

I myself use and abuse "you" in documentation. But this is generally not useful.

**Review by groue — CHANGES_REQUESTED** [review]
Thank you very much! Putting SPM first is a good idea 👍 

There a few wordings and formatting to enhance.

File: README.md (hunk 1 above)
Comment (SagarSDagdu) [reply README.md]: Moved it to its original place 👍🏻

File: README.md (hunk 3 above)
Comment (SagarSDagdu) [reply README.md]: Added indentation

File: README.md (hunk 5 above)
Comment (SagarSDagdu) [reply README.md]: 😄 changed it

File: README.md (hunk 4 above)
Comment (SagarSDagdu) [reply README.md]: Removed

File: README.md [hunk 6]
```diff
@@ -350,15 +361,21 @@ pod 'GRDB.swift'
...
+```ruby
+    # Can't use semantic versioning due to https://github.com/CocoaPods/CocoaPods/issues/11839
+    pod 'GRDB.swift', git: 'https://github.com/groue/GRDB.swift.git', branch: 'GRDB6'
+```
+- Depend on a specific version explicitly (Replace the tag with the version you want to use):
+```ruby
+    # Can't use semantic versioning due to https://github.com/CocoaPods/CocoaPods/issues/11839
+    # Replace the tag with the tag that you want to use.
+    pod 'GRDB.swift', git: 'https://github.com/groue/GRDB.swift.git', tag: 'v6.29.0' 
+```
```
Comment (groue) [inline README.md:378]: 😉 This is not how Markdown indentation works. Here is the correct solution (and I also added some breathing with blank lines):

```suggestion
- Depend on the `GRDB6` branch. This is more or less equivalent to what `pod 'GRDB.swift', '~> 6.0'` would normally do, if CocoaPods would accept new GRDB versions to be published:

    ```ruby
    # Can't use semantic versioning due to https://github.com/CocoaPods/CocoaPods/issues/11839
    pod 'GRDB.swift', git: 'https://github.com/groue/GRDB.swift.git', branch: 'GRDB6'
    ```

- Depend on a specific version explicitly (Replace the tag with the version you want to use):

    ```ruby
    # Can't use semantic versioning due to https://github.com/CocoaPods/CocoaPods/issues/11839
    # Replace the tag with the tag that you want to use.
    pod 'GRDB.swift', git: 'https://github.com/groue/GRDB.swift.git', tag: 'v6.29.0' 
    ```
```

**Review by groue — APPROVED** [review]
LGTM! Thank you very much @SagarSDagdu 🙏

**Comment by groue** [comment]
I'll soon push to the master branch. Thanks again for reminding that the situation was in a sad state 👍

**Comment by groue** [comment]
Shipped in 6.29.1. Thank you!

**Pull request merged**